app/controllers/comments_controller.rb

```diff
@@ -50,6 +53,7 @@ def update
 
   # DELETE /comments/1 or /comments/1.json
   def destroy
+    authorize @comment
```

Comment:
We can remove this now since we already call it everywhere with the `before_action`